SUMMARY.md

```diff
@@ -110,6 +110,10 @@
     - [Encryption / Decryption](developers/provider/encryption-decryption.md)
     - [Compute Endpoints](developers/provider/compute-endpoints.md)
```

Comment:
The uploader files are not visible inside the user guides as they are not linked here. Please update the SUMMARY.md to include them as well. Thanks 📙